ui/admin/app/templates/scopes/scope/host-catalogs/host-catalog/host-sets/host-set/hosts/index.hbs

```diff
@@ -31,10 +31,14 @@
       <Rose::Table as |table|>
         <table.header as |header|>
           <header.row as |row|>
-            <row.headerCell>{{t 'form.name.label'}}</row.headerCell>
+            {{#if @model.hostSet.isStatic}}
+              <row.headerCell>{{t 'form.name.label'}}</row.headerCell>
+            {{else}}
+              <row.headerCell>{{t 'form.external_name.label'}}</row.headerCell>
+            {{/if}}
             <row.headerCell>{{t 'form.type.label'}}</row.headerCell>
             <row.headerCell>{{t 'form.id.label'}}</row.headerCell>
-            {{#if @model.isStatic}}
+            {{#if @model.hostSet.isStatic}}
```

Comment:
I think this was a bug since it wasn't ever displaying the actions header

Reply:
Non-blocking question: What was the reason for having `@model.isStatic` in the first place since it seems like hosts belonging to a static host-catalog are listed in this [file ](https://github.com/hashicorp/boundary-ui/blob/main/ui/admin/app/templates/scopes/scope/host-catalogs/host-catalog/hosts/index.hbs) (in [vercel](https://boundary-ui-git-icu-8795-dynamic-host-catalog-0d4d11-hashicorp.vercel.app/scopes/s_1sheljmjgm/host-catalogs/hc_pkq4tp9lzg/hosts) )?

Reply:
Ah I def. misunderstood! Thanks for clarifying!